test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1

```diff
@@ -59,3 +59,37 @@ Describe "Add-Content cmdlet tests" -Tags "CI" {
     }
   }
 }
+
+Describe "Add-Content feature tests" -Tag Feature {
+  It "Should not block reads while writing" {
+    $logpath = Join-Path $testdrive "test.log"
+
+    $writer = Start-Job -ScriptBlock {
+      param($logpath)
+
+      1..10 | % {
```

Comment:
Use `ForEach-Object` instead of `%`

Reply:
is `(1..10).foreach{...}` faster then `ForEach-Object`?